package.json

```diff
@@ -2,11 +2,20 @@
   "name": "@bigbinary/neeto-hotkeys",
   "version": "1.0.0",
   "description": "React hook for handling keyboard shortcuts, offering a customizable and platform-aware solution, supporting global and scoped modes.",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "export": "./dist/index.cjs.js"
+    }
+  },
   "scripts": {
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "build": "NODE_ENV=production rollup -c rollup.config.js --bundleConfigAsCjs"
```

Comment:
What is `bundleConfigAsCjs` for?

Reply:
It basically force the configuration to be transpiled to CommonJS even if the configuration itself is written as an ES module. I was getting an error as: `error: Cannot use import statement outside a module`. And it was suggested in the error to add `bundleConfigAsCjs` flag to transpiled to CommonJS.

<img width="1280" alt="Screenshot 2023-11-17 at 11 21 53 AM" src="https://github.com/bigbinary/neeto-hotkeys/assets/67572237/a53272be-6719-4dea-84c4-d61d610f506b">